src/ui/server/api/keystone/application/update/exportContractAgent.ts

```diff
@@ -0,0 +1,43 @@
+import apollo from '../../../../graphql/apollo';
+import updateApplicationExportContractAgentMutation from '../../../../graphql/mutations/update-application/export-contract-agent';
+import { ApolloResponse } from '../../../../../types';
+
+/**
+ * updateExportContractAgent
+ * Update an application's "exportContractAgent"
+ * @param {String} ExportContractAgent ID
+ * @param {Object} ExportContractAgent update
+ * @returns {Object} Updated export contract agent
+ */
+const updateExportContractAgent = async (id: string, update: object) => {
```

Comment:
`object` is very generic type, doesn't lint complain about it?
is it easy to describe it?

Reply:
Excellent shout thanks @avaitonis , we have this in quite a few places, i'm updating all of them 👍

Reply:
maybe also type for errorList

Reply:
Updated 😊  https://github.com/UK-Export-Finance/exip/pull/2195/commits/463909bdc9d6675888b229d551f48e1ecc53102e

Reply:
@avaitonis documentation for `errorList` would be good but very fiddly as it's different for each time it's used, e.g

```js
errorList: {
  fieldA: { text }, // only applicable to form X
  fieldB: { text },  // only applicable to form Y
  fieldC: { text },  // applicable to form X and Y
  // etc
}
```

We could add types for this, but I would prefer to keep it as a generic `object`, to reduce overhead. It could easily become out of sync if the possible errors is changed. What do you think?